pkg/collector/corechecks/cluster/orchestrator/processors/k8s/serviceaccount.go

```diff
@@ -97,3 +97,9 @@ func (h *ServiceAccountHandlers) ScrubBeforeExtraction(ctx *processors.Processor
 // it is marshalled to generate a manifest.
 func (h *ServiceAccountHandlers) ScrubBeforeMarshalling(ctx *processors.ProcessorContext, resource interface{}) {
 }
+
+// BuildManifestMessageBody is used to build a message containing a chunk of
+// manifest models of a certain size.
+func (h *ServiceAccountHandlers) BuildManifestMessageBody(ctx *processors.ProcessorContext, resourceManifests []interface{}, groupSize int) model.MessageBody {
```

Comment:
Does this needs to be part of the method and implemented for every resource? For me, it seems it can be a function which is called from the processor loop. Except there is something unique per resource i.e. object parsing etc.

Reply:
I wanted to make it be consistent because we have `BuildMessageBody` for metadata. But indeed, it's not worthy to keep it as it increase the redundancy.

 A common function is added :
  https://github.com/DataDog/datadog-agent/blob/551ba56141bb206e8be1dd6021425594f156abf6/pkg/collector/corechecks/cluster/orchestrator/processors/processor.go#L199-L213